### ISSUE TITLE
yrm ls throw error

### ISSUE THREAD

**Issue by reaink** [YunYouJun/nnrm#10]
<img width="823" alt="image" src="https://user-images.githubusercontent.com/45023042/190131223-a922aa41-26a6-41d3-9fa2-654b40d75dc9.png">
```
Error: Command failed with exit code 1: yarn config get registry
Usage Error: Couldn't find a configuration settings named "registry"
```

**Comment by YunYouJun** [comment]
Yrm is used to switch `yarn` registries. Are you sure you have `yarn` installed?

You can use `nnrm` or `nrm` to change `npm/pnpm` registries.

**Comment by reaink** [comment]
@YunYouJun installed yarn

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/45023042/190300409-dd2c897a-a972-4f9d-9cbe-fdef08662509.png">
log:

``` bash
Error: Command failed with exit code 1: yarn config get registry
Usage Error: Couldn't find a configuration settings named "registry"

$ yarn config get [--json] [--no-redacted] <name>
    at makeError (file:///Users/rea-work/Library/pnpm/global/5/.pnpm/execa@6.1.0/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///Users/rea-work/Library/pnpm/global/5/.pnpm/execa@6.1.0/node_modules/execa/index.js:119:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getCurrentRegistry (file:///Users/rea-work/Library/pnpm/global/5/.pnpm/nnrm@0.3.6/node_modules/nnrm/utils/index.js:69:22)
    at async listRegistries (file:///Users/rea-work/Library/pnpm/global/5/.pnpm/nnrm@0.3.6/node_modules/nnrm/utils/index.js:52:27)
    at async CAC.onLs (file:///Users/rea-work/Library/pnpm/global/5/.pnpm/nnrm@0.3.6/node_modules/nnrm/utils/index.js:139:5) {
  shortMessage: 'Command failed with exit code 1: yarn config get registry',
  command: 'yarn config get registry',
  escapedCommand: 'yarn config get registry',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: `\x1B[31m\x1B[1mUsage Error\x1B[22m\x1B[39m: Couldn't find a configuration settings named "registry"\n` +
    '\n' +
    '\x1B[1m$ \x1B[22myarn config get [--json] [--no-redacted] <name>',
  stderr: '',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

**Comment by YunYouJun** [comment]
The latest version of yarn is `1.22.19`.

Is the yarn you installed really the YARN we mean?

yarnpkg.com

**Comment by reaink** [comment]
> The latest version of yarn is `1.22.19`.
> 
> Is the yarn you installed really the YARN we mean?
> 
> yarnpkg.com

Yes, it is yarn, But i set `yarn set version stable` use 3.2.3 version.

<img width="694" alt="image" src="https://user-images.githubusercontent.com/45023042/190343855-088d5146-2d31-4f53-aee5-604d4e3419fd.png">

**Comment by YunYouJun** [comment]
OK.

Yarn 3 seems have a break change.

They do not support `yarn config get registry`, and we can use `yarn config get npmRegistryServer`,
I will fix it by judging yarn version.

**Comment by YunYouJun** [comment]
One more query is required when judging version, so I'm going to use `try/catch` directly to add it as a fallback.

**Comment by YunYouJun** [comment]
https://github.com/YunYouJun/nnrm/commit/cbb1ace7333f7d03af9483e47e90e9f1ca2a908d

**Comment by YunYouJun** [comment]
Released in `v0.3.7`.
